sync/category_converter.go

```diff
@@ -0,0 +1,86 @@
+package sync
+
+import (
+	"database/sql"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/appuio/appuio-cloud-reporting/pkg/db"
+	"github.com/vshn/appuio-odoo-adapter/odoo/model"
+)
+
+const (
+	elementSeparator = ":"
+)
+
+// invoiceCategoryNameRegex parses a category name in the form of "prefix Zone: zone - Namespace: namespace"
+var invoiceCategoryNameRegex = regexp.MustCompile("Zone: (\\w+) - Namespace: (\\w+)")
+
+// CategoryConverter converts category models to and from Odoo.
+type CategoryConverter struct{}
```

Comment:
Why does this struct exist? Why are the conversion functions methods and not functions?

Reply:
Personal preference :shrug:  I'm not that big fan of bunch of static functions in a package